scripting/get5/stats.sp

```diff
@@ -296,6 +329,54 @@ public Action Stats_PlayerDeathEvent(Event event, const char[] name, bool dontBr
     g_RoundClutchingEnemyCount[clutcher] = tCount;
   }
 
+  int mapNumber = GetMapNumber();
+  int roundTime = GetMilliSecondsPassedSince(g_RoundStartedTime);
+  int penetrated = event.GetInt("penetrated");
+  bool thruSmoke = event.GetBool("thrusmoke");
+  bool attackerBlind = event.GetBool("attackerblind");
+  bool noScope = event.GetBool("noscope");
+  bool friendlyFire = validAttacker ? attackerTeam == victimTeam : false;
+  bool assistFriendlyFire = validAssister ? assisterTeam == victimTeam : false;
+
+  EventLogger_PlayerDeath(
+    g_RoundNumber,
+    roundTime,
+    validAttacker ? attacker : 0,
+    victim, // we already checked that victim is valid.
+    headshot,
+    validAssister ? assister : 0,
+    assistedFlash,
+    weapon,
+    friendlyFire,
+    assistFriendlyFire,
+    penetrated,
+    thruSmoke,
+    noScope,
+    attackerBlind
+  );
+
+  LogDebug("Calling Get5_OnPlayerDied(weapon=%s, headshot=%d, mapNumber=%d, roundNumber=%d, roundTime=%d, attacker=%d, victim=%d, assister=%d, assistedFlash=%d, penetrated=%d, thruSmoke=%d, noScope=%d, attackerBlind=%d, attackerTeam=%d, assisterTeam=%d, victimTeam=%d)",
```

Comment:
Maybe make the event `Get5_OnPlayerDeath`?

Reply:
+1